site/docs/components/icons.md

```diff
@@ -1,80 +1,58 @@
 ---
 layout: docs
-title: Icons (FloSports) (WIP)
-description: Use SVG Icons in HTML
+title: Icons
+description: SVG Icons in HTML
 group: components
 toc: true
 ---
-# NOT Updated for FLO-SCSS yet
 
-## Icons
+We use svgs in the design system and not a font library to display icons. Look here for details on [Brand Logos]({{ site.baseurl }}/docs/components/logos) or [Badges]({{ site.baseurl }}/docs/components/badge).
+
+## Add SVG
+- Step 1
+- step 2
+- step 3
+
+ 
+## Manipulating SVG's
 
-### Size
+### Sizing
+Icons inherit width and height from the parent container, so the parent container can set its dimensions.  
 
```

Comment:
I would add a code snippet to show an example of size on the parent container, but we don't need to paste in an entire inline svg. Something similar to this but with the correct classes
```
<div class="row row-cols-4 align-items-center">   // <- reference size in parent
   <svg class="icon-sm">
```
Thoughts?

Reply:
also I would change it to "so the parent container can set the **SVG's** dimensions"